### PR TITLE
Allow :Buffers to take a query argument

### PR DESCRIPTION
**Pull request by tbelaire**
This works just like how :Lines does, except I also allow it to
autocomplete based on buffers.

### CODE REVIEW

File: plugin/fzf.vim [hunk 1]
```diff
@@ -38,27 +38,27 @@ function! s:defs(commands)
...
-\'command! -bar -bang Windows                       call fzf#vim#windows(<bang>0)',
-\'command! -bar -bang Commits                       call fzf#vim#commits(<bang>0)',
-\'command! -bar -bang BCommits                      call fzf#vim#buffer_commits(<bang>0)',
-\'command! -bar -bang Maps                          call fzf#vim#maps("n", <bang>0)',
-\'command! -bar -bang Filetypes                     call fzf#vim#filetypes(<bang>0)',
-\'command!      -bang -nargs=* History              call s:history(<q-args>, <bang>0)'])
+\'command!      -bang -nargs=? -complete=dir Files       call fzf#vim#files(<q-args>, <bang>0)',
+\'command!      -bang -nargs=? GitFiles                  call fzf#vim#gitfiles(<q-args>, <bang>0)',
+\'command!      -bang -nargs=? GFiles                    call fzf#vim#gitfiles(<q-args>, <bang>0)',
+\'command!      -bang -nargs=? -complete=buffer Buffers  call fzf#vim#buffers(<q-args>, <bang>0)',
```
Comment (junegunn) [inline plugin/fzf.vim]: Let's keep `-bar` here.

Comment (tbelaire) [inline plugin/fzf.vim]: Oops, missed that when I rebased.  Good catch, thanks.

**Comment by tbelaire** [comment]
Oops, does squashing the changes right away ruin the github review comments? Sorry.

Are you sure you want "-bar", even if it takes an argument now?

**Comment by junegunn** [comment]
It's unlikely that we'll be typing in bar characters in the argument for Buffers command, so yes.
Merged, thanks!

**Pull request merged**